specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/ManagedInstanceLongTermRetentionPolicies.json

```diff
@@ -301,6 +278,41 @@
           "readOnly": true
         }
       }
+    },
+    "ManagedInstanceLongTermRetentionPolicyProperties": {
+      "description": "Properties of a long term retention policy",
+      "type": "object",
+      "properties": {
+        "backupStorageAccessTier": {
+          "description": "The BackupStorageAccessTier for the LTR backups",
+          "enum": [
+            "Hot",
+            "Archive"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "BackupStorageAccessTier",
+            "modelAsString": true
+          }
+        },
+        "weeklyRetention": {
```

Comment:
>weeklyRetention [](http://example.com/codeflow?start=9&length=15)

What are the possible values for these Retention properties?

Reply:
Should this be an enum? Also pl provide more details in the description

Reply:
The possible values are: 1-3650 days, 1-520 weeks or 1-10years (annotations is for e.g. P43D. ) so enum is not possible.
The retention is fully documented in: https://learn.microsoft.com/en-us/azure/azure-sql/database/long-term-retention-overview?view=azuresql

Reply:
It would be good to callout the format of the retention format (like you mentioned ISO duration format) here.